### PR TITLE
Fix `classification` notebook to download `FordA` dataset without error

### PR DESCRIPTION
**Pull request by Mr-Geekman**
## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Proposed Changes
Rewrite passage for downloadig `FordA` dataset.

### CODE REVIEW

File: examples/classification.ipynb [hunk 1]
```diff
@@ -5,7 +5,11 @@
    "id": "cb9e5d62",
```
Comment (brsnw250) [inline examples/classification.ipynb:5]: <p>Maybe rephrase the second paragraph to make it simpler. Like "It is possible to load ... . But here we show how to do it manually". </p><p> </p><p>Also, should it be <em>from</em> instead of <em>form</em>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (brsnw250) [inline examples/classification.ipynb:5]: <p>Maybe remove the second word <em>methods </em>in the&nbsp;second paragraph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (brsnw250) [inline examples/classification.ipynb:5]: <p>List markdown is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Review by brsnw250 — CHANGES_REQUESTED** [review]
Small fixes

File: examples/classification.ipynb (hunk 1 above)
Comment (Mr-Geekman) [reply examples/classification.ipynb:5]: <p>It renders fine on github and locally.</p>

Comment (brsnw250) [reply examples/classification.ipynb:5]: <p>Maybe remove the reasoning why we don't use fetch_ucr_dataset. Just note that it is possible to do it with this function, but we load data manually.</p><p>This part may become irrelevant if we consider the possibility of changing the version in the future.</p>

Comment (brsnw250) [reply examples/classification.ipynb:5]: <p>Ok, so let's leave it as is, I guess.</p>

**Review by brsnw250 — DISMISSED** [review]
